features/torque2/client/config.pan

```diff
@@ -41,8 +41,8 @@ variable TORQUE_TMPDIR = if ( exists(TORQUE_TMPDIR) &&
 # This is done to avoid potential error messages on Torque master when
 # client tries to contact server.
 "/software/components/chkconfig/service/pbs_mom" = {
-  wn_list = nlist();
-  if ( exists(WORKER_NODES_NLIST[FULL_HOSTNAME]) || TORQUE_FORCE_MOM_STARTUP ) {
+  wn_list = dict();
+  if ( exists(WORKER_NODES_DICT[FULL_HOSTNAME]) || TORQUE_FORCE_MOM_STARTUP ) {
```

Comment:
Will renaming `WORKER_NODES_NLIST` to `WORKER_NODES_DICT` break backwards compatibility with existing templates?
If so, this should be specifically mentioned in the PR description and the PR should be marked as "backwards incompatible".

Reply:
You are right. I forgot to push another updates. The issue is coming from the fact that UMD-3 and UMD-4 have diverged (only UMD-3 templates have been updated!). UMD-3 should now be considered as 'deprecated' and we have to report our efforts on UMD-4. Lots of work to do...

Reply:
Yikes. Thanks for working on this, much appreciated!
Do you think the default branch on GitHub should be updated to `umd-4` once this is merged?

Reply:
I have first to compare the two branches before making umd-4 as default branch.